dist/tools/pyterm/pyterm

```diff
@@ -105,7 +105,7 @@ class SerCmd(cmd.Cmd):
     def __init__(self, port=None, baudrate=None, toggle=None, tcp_serial=None,
                  confdir=None, conffile=None, host=None, run_name=None,
                  log_dir_name=None, newline=None, formatter=None,
-                 serprompt=None):
+                 invertRTS=None, invertDTS=None, serprompt=None):
```

Comment:
Maybe change the name to setRTS and setDTS then a number to make it more generic since invert seems to set RTS to 0 but DTS to 1.

Reply:
Yeah, that's better. Changed to that approach. Thanks!